src/Client.js

```diff
@@ -1,20 +1,22 @@
+import { withRootUrl } from 'taskcluster-lib-urls';
 import { stringify } from 'query-string';
 import hawk from 'hawk';
 import fetch from './fetch';
 
-const REMOVE_TRAILING_SLASH = /\/$/;
-
 export default class Client {
   static defaults = {
+    rootUrl: process.env.TASKCLUSTER_ROOT_URL,
```

Comment:
does this make sense in a web context?  I kind of think there should be no default.